apiserver/plane/utils/issue_filters.py

```
@@ -622,6 +622,89 @@
+                if direction == "after":
+                    issue_filter[f"{field}__gte"] = ts
+                elif direction == "before":
+                    issue_filter[f"{field}__lte"] = ts
+            else:
+                issue_filter[f"{field}__exact"] = ts
+    return issue_filter
+
+
+def filter_created_at_ts(params, issue_filter, method, prefix=""):
+    """
+    Applies timestamp-level filtering to the created_at field based on the 'created_at_ts' parameter.
```

Pylint (3.3.7)

[convention] Line too long (101/100)

(C0301)